pre_install_report/library/pre_install_check_permissions.py

```diff
@@ -42,6 +45,8 @@
 PROVISIONER_AZURE_DISK = "kubernetes.io/azure-disk"
 PROVISIONER_AWS_EBS = "kubernetes.io/aws-ebs"
 
+INGRESS_REL = '<1.19'
```

Comment:
This is later used to decide whether a "v118" named hello-ingress yaml file.    What if current_version (see var used later) is not v1.18?   Is it possible we have any 1.17 env where this would execute??

Reply:
Spoke with @erharb and he pointed me to the current documentation.  We have to support 1.18 till the next LTS.  Som instead of checking for <1.19 for backward compatibility, I will check specifically for 1.18